### PR TITLE
added LocationSample.

### PR DESCRIPTION
**Pull request by teamagam1**
it is immutable and holds valuable data from Location instance.

PointGeometry has hasAltitude field.

### CODE REVIEW

File: app/src/main/java/com/teamagam/gimelgimel/app/model/entities/LocationSample.java [hunk 1]
```diff
@@ -1,31 +1,208 @@
...
         mPoint = point;
         mTime = time;
-        mUserId = userId;
     }
 
-    //get + set
+    /**
+     * Construct a new Location Sample that is copied from an existing Location.
+     */
+    public LocationSample(Location l) {
```
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/model/entities/LocationSample.java:62]: Usually, constructor doesn't implement any logic/parsing.
When the need arises, a factory method can be used.

File: app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/data/geometries/PointGeometry.java [hunk 2]
```diff
@@ -16,7 +16,10 @@
     public double longitude;
 
     @SerializedName("altitude")
-    public double altitude;
+    public double altitude = 0.0f;
```
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/data/geometries/PointGeometry.java]: member is initialize in constructor !fix

File: app/src/main/java/com/teamagam/gimelgimel/app/model/entities/LocationSample.java (hunk 1 above)
Comment (teamagam1) [inline app/src/main/java/com/teamagam/gimelgimel/app/model/entities/LocationSample.java:62]: !completed

File: app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/data/geometries/PointGeometry.java (hunk 2 above)
Comment (teamagam1) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/data/geometries/PointGeometry.java]: !fixed

**Pull request merged**